### PR TITLE
Documentaion for java intergration with etcd

### PR DESCRIPTION
**Pull request by sarvaniy**
Signed-off-by: sarvaniy <sarvanimounika@gmail.com>

### CODE REVIEW

File: projects/etcd/java/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# java integration with etcd
```
Comment (ChaitanyaKaranam) [inline projects/etcd/java/README.md]: # Java integration with etcd

Comment (ChaitanyaKaranam) [inline projects/etcd/java/README.md:1]: Few nitpicks:

``Java integration with etcd``

File: projects/etcd/java/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    -p 2379:2379 \
+    --restart=always \
+    --name etcd gcr.io/etcd-development/etcd:latest \
+    /usr/local/bin/etcd \
+    --data-dir=/etcd-data --name <hostname> \
+    --advertise-client-urls http://<host_ip>:2379 --listen-client-urls http://0.0.0.0:2379 \
+    --heartbeat-interval 1000 \
+    --election-timeout 5000
+```
+## java Integration
```
Comment (ChaitanyaKaranam) [inline projects/etcd/java/README.md:18]: ``Java Integration``

File: projects/etcd/java/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+### Download
+  #### Maven
+```
+      <dependency>
+        <groupId>io.etcd</groupId>
+        <artifactId>jetcd-core</artifactId>
+        <version>${jetcd-version}</version>
+      </dependency>
+```
```
Comment (tandonri) [inline projects/etcd/java/README.md:35]: 1. Which nexus repo will one find this artifact.
2. what version was used in your example.

Comment (sarvaniy) [reply projects/etcd/java/README.md:35]: Hi Rishi,

1. Maven Central will have this artifact.
    https://repo.maven.apache.org/maven2/io/etcd/jetcd-core/0.4.1/jetcd-core-0.4.1.jar
2. used version is 0.4.1

**Pull request merged**